src/main/resources/templates/index.html

```diff
@@ -7,6 +7,9 @@
 
 </head>
 <body class="dashboard dashboard_2">
+
+<div th:replace="layout/include_scripts"></div>
```

Comment:
can't we put js at bottom of the page to speed up page loading ?

Reply:
Not yet.
We have some inline JS on a lot of pages, that require '$' or 'Vue' (we're not on window.onload() callbacks :cry: ).
So I had to put this in top of our pages.
Maybe we should open a separate issue for this cas.